app/src/main/java/com/litewallet/di/Module.kt

```diff
@@ -0,0 +1,19 @@
+package com.litewallet.di
+
+import com.breadwallet.tools.manager.BRApiManager
+import com.google.firebase.ktx.Firebase
+import com.google.firebase.remoteconfig.ktx.remoteConfig
+import com.litewallet.data.source.RemoteConfigSource
+
+class Module(
```

Comment:
this is for dependency injection, since in the future we want to migrate into kotlin, for now we just add this module class, later we will using [Koin](https://insert-koin.io/)

Reply:
It's kotlin dependency injection framework.

In short, dependency injection framework like dagger/hilt

Reply:
Good

Reply:
Good to think of the future @andhikayuana .

Reply:
sure, since in the future we plan to use kotlin, will revisit this.
as for now, we just using "manual" dependency injection